test/test.js

```diff
@@ -95,5 +95,16 @@ describe('@rdfjs/parser-n3', () => {
       strictEqual(typeof error, 'object')
       strictEqual(error.message.includes('literal'), true)
     })
+
+    it('should forward options to n3 parser', async () => {
```

Comment:
Can you change it to *should forward blankNodePrefix* option to n3 parser*?

Reply:
Of course, done